docker/Dockerfile.backend

```diff
@@ -1,11 +1,12 @@
 FROM python:3.9.10-buster
 
-WORKDIR /app
+WORKDIR /app/backend/
 COPY backend/ /app
 
 ENV PATH="$PATH:/root/.local/bin"
 
 RUN curl -sSL https://install.python-poetry.org | python3 - && \
-    poetry install
+    poetry install && \
+    poetry config virtualenvs.create false
```

Comment:
Please reverse the order for this call
It should look like this:
```bash
RUN curl -sSL https://install.python-poetry.org | python3 - \
    && poetry config virtualenvs.create false \
    && poetry install
```

Reply:
Done, fixed call order